test/NuGet.Core.Tests/NuGet.CommandLine.Xplat.Tests/MSBuildAPIUtilityTests.cs

```diff
@@ -0,0 +1,384 @@
+</Project>";
+            File.WriteAllText(Path.Combine(testDirectory, "projectA.csproj"), projectContent);
+
+            var project = Project.FromFile(Path.Combine(testDirectory, "projectA.csproj"), projectOptions);
+
+            var result = new MSBuildAPIUtility(logger: new TestLogger()).GetDirectoryBuildPropsRootElement(project);
+
+            Assert.Equal(Path.Combine(testDirectory, "Directory.Packages.props"), result.FullPath);
+        }
+
+        [Fact]
+        public void AddPackageReferenceIntoProjectFileWhenItemGroupDoesNotExist_Success()
+        {
+            // Set up
+            var testDirectory = TestDirectory.Create();
+            var projectCollection = new ProjectCollection(
+                            globalProperties: null,
+                            remoteLoggers: null,
+                            loggers: null,
+                            toolsetDefinitionLocations: ToolsetDefinitionLocations.Default,
+                            // Having more than 1 node spins up multiple msbuild.exe instances to run builds in parallel
+                            // However, these targets complete so quickly that the added overhead makes it take longer
+                            maxNodeCount: 1,
+                            onlyLogCriticalEvents: false,
+                            loadProjectsReadOnly: false);
+
+            var projectOptions = new ProjectOptions
+            {
+                LoadSettings = ProjectLoadSettings.DoNotEvaluateElementsWithFalseCondition,
+                ProjectCollection = projectCollection
+            };
+
+            // Set up project file
+            string projectContent =
+@$"<Project Sdk=""Microsoft.NET.Sdk"">
+<PropertyGroup>                   
+<TargetFramework>net6.0</TargetFramework>
+</PropertyGroup>
+</Project>";
+            File.WriteAllText(Path.Combine(testDirectory, "projectA.csproj"), projectContent);
+            var project = Project.FromFile(Path.Combine(testDirectory, "projectA.csproj"), projectOptions);
+
+            var msObject = new MSBuildAPIUtility(logger: new TestLogger());
+            // Creating an item group in the project
+            var itemGroup = msObject.CreateItemGroup(project, null);
+
+            var libraryDependency = new LibraryDependency
+            {
+                LibraryRange = new LibraryRange(
+                        name: "X",
+                        versionRange: VersionRange.Parse("1.0.0"),
+                        typeConstraint: LibraryDependencyTarget.Package)
+            };
+
+            // Act
+            msObject.AddPackageReferenceIntoItemGroupCPM(project, itemGroup, libraryDependency);
+            project.Save();
+
+            // Assert
+            Assert.Contains(@$"<PackageReference Include=""X"" />", File.ReadAllText(Path.Combine(testDirectory, "projectA.csproj")));
+            Assert.DoesNotContain(@$"<Version = ""1.0.0"" />", File.ReadAllText(Path.Combine(testDirectory, "projectA.csproj")));
```

Comment:
AFAIK, Version will not be added as a child element to the `PackageReference` item.  The format will be `<PackageReference Include="Newtonsoft.Json" Version="13.0.2-beta1" />`

```suggestion
Assert.DoesNotContain(@$"<PackageReference Include=""X"" Version = ""1.0.0"" />", File.ReadAllText(Path.Combine(testDirectory, "projectA.csproj")));
```
This suggestion applies to other tests that assert for `Version` attribute.

Reply:
Agreed, will fix for all the tests. Thanks for the catch!